src/components/auth/Register.js

```diff
@@ -0,0 +1,411 @@
+
+                                
+                                <Form.Group>
+                                    <Form.Control
+                                        value={occupation}
+                                        onChange={(event) => setOccupation(event.target.value)}
+                                        placeholder="Occupation"
+                                    />
+                                    <Form.Text className="text-muted"></Form.Text>
+                                </Form.Group>
+
+
+                                <Form.Group>
+                                    <Form.Row>
+                                        <Col>
+                                        <Form.Control
+                                            value={licenseType}
+                                            onChange={(event) => setLicenseType(event.target.value)}
+                                            placeholder="License Type"
+                                        />
+                                        </Col>
+                                        <Col>
+                                        <Form.Control
+                                            value={license}
+                                            onChange={(event) => setLicense(event.target.value)}
+                                            placeholder="License #"
+                                        />
+                                        </Col>
+                                        <Form.Text className="text-muted"></Form.Text>
+                                    </Form.Row>
+                                </Form.Group>
+                                <Stepper steps={steps} activeStep={currentStep}/>
+                                <Form.Row>
+                                    <Col>
+                                        <Button
+                                            className="back-button"
+                                            onClick={previousStep}
+                                            block
+                                        >
+                                            Back
+                                        </Button>
+                                    </Col>
+                                    <Col>
+                                        <Button
+                                            className="next-button"
+                                            onClick={handleSubmit}
+                                            block
+                                        >
+                                            Submit
+                                        </Button>
+                                    </Col>
+                                </Form.Row>
+                            </Form>
+                        }
+                    </div>
+                </Container>
+            </div>
+        </Styles>
+    );
+};
+
+export default Register;
```

Comment:
Like `Pages/Signup.js`, this file's indentation is 4 spaces. The rest of the project is 2 spaces. Idc which standard we go with but we should be consistent across all the files